tests/conftest.py

```diff
@@ -15,14 +57,12 @@ def collect(self):
         test_dir = os.path.dirname(str(self.fspath))
         for platform, archs in spec["Output Files"].items():
             for arch, filename in archs.items():
-                filepath = os.path.join(test_dir, filename)
-                if os.path.exists(filepath):
-                    yield FLOSSTest(self, platform, arch, filename, spec)
-
-
-def extract_strings(sample_path):
-    # TODO: implement me
-    return []
+                # TODO only test Windows PE files for now
+                _, fileext = os.path.splitext(filename)
+                if fileext == ".exe":
+                    filepath = os.path.join(test_dir, "bin", filename)
+                    if os.path.exists(filepath):
+                        yield FLOSSTest(self, platform, arch, os.path.join("bin", filename), spec)
```

Comment:
i think `test.yml` should indicate the output directory and path. assuming there's a `bin` directory is probably usually correct, but should be configurable.